From e1e0c13da9654081a46222836716eff73aed8c8d Mon Sep 17 00:00:00 2001
From: Andrew Savchenko <bircoph@gentoo.org>
Date: Sun, 13 Sep 2020 10:15:51 +0300
Subject: [PATCH 12/14] xbsa

When xbsa (tsm) support is enabled, butxInfo is defined in the
butc_xbsa.h, so it should not be redefined here, otherwise multiple
definitions of the butxInfo symbol error will occur.

(cherry picked from commit 032844997f8c57168b91550c929ac1dd6544bafa)
---
 src/butc/lwps.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/butc/lwps.c b/src/butc/lwps.c
index 3078d7382..9769862bf 100644
--- a/src/butc/lwps.c
+++ b/src/butc/lwps.c
@@ -53,7 +53,7 @@ extern int forcemultiple;
 /* XBSA Global Parameters */
 extern afs_int32 xbsaType;
 #ifdef xbsa
-struct butx_transactionInfo butxInfo;
+extern struct butx_transactionInfo butxInfo;
 #endif
 
 static struct TapeBlock {		/* A 16KB tapeblock */
-- 
2.34.1

